ocs_ci/ocs/resources/pvc.py

```diff
@@ -162,6 +162,26 @@ def resize_pvc(self, new_size, verify=False):
             )
         return True
 
+    def get_attached_pods(self):
+        """
+        Get the pods attached to the PVC represented by this object instance
+
+        Returns:
+            list: A list of pod objects attached to the PVC
+
+        Note:
+             # Importing from pod inside, because of unsolvable import loop
+             from ocs_ci.ocs.resources.pod import get_all_pods, get_pvc_name
+
+        """
+        from ocs_ci.ocs.resources.pod import get_all_pods, get_pvc_name
+        attached_pods = []
+        all_pods = get_all_pods()
+        for pod_obj in all_pods:
```

Comment:
Can be reduced to `for pod_obj in get_all_pods()`, I don't think a variable is needed

Reply:
Can also be highly compacted with some list comprehension - 
`return [pod_obj for pod_obj in get_all_pods() if get_pvc_name(pod_obj) == self.name]`

Reply:
made it long for readability :)